render/meson.build

```diff
@@ -14,13 +14,10 @@ wlr_files += files(
 	'wlr_texture.c',
 )
 
-egl = dependency('egl', required: 'gles2' in renderers)
-if egl.found()
+if 'gles2' in renderers or 'auto' in renderers
+	egl = dependency('egl')
```

Comment:
Hm, this isn't quite right. If running with `auto`, we'll want to look for the `egl` dep, and disable the `gles2` renderer if it's missing. Here the `egl` dep is mandatory.